docs/products/eigencompute/build-app-eigencompute.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Build a Verifiable App
+sidebar_position: 4
+---
+
+The steps to build a verifiable app on EigenCompute are shown below.
+
+<img src="/img/devJourneyEigenCompute.png" alt="EigenCompute Developer Journey"/>
+
+For more information, see the following topics:
+
+1. Create Authentication Keys 
+2. Create an App 
```

Comment:
`eigenx app create` is sort of an optional command. if the developer already has a docker image and env file, they can deploy it directly, rather than starting from a template.